pyne/utils.py

```diff
@@ -74,7 +83,11 @@ def to_sec(input_time, units):
         Time value in [sec].
 
     """
-    sec_time = input_time * time_conv_dict[units.lower()]
+    units = units.lower()
+    try:
+        sec_time = time_conv_dict[units] * input_time
+    except KeyError:
```

Comment:
Exceptions should only be used for exceptional cases.  Please check containment with `key in dict` or use `dict.get()`